book/objects.tex

```diff
@@ -1791,7 +1791,7 @@ \subsection{Constructing Objects with Private Attributes}
 \index{attribute!private}
 \index{attribute!public}
 In this example, we have declared \verb'$.x' and \verb'$.y' 
-as ``public'' (so to speak) attributes, and \verb'$.z' as 
+as ``public'' (so to speak) attributes, and \verb'$!z' as a
```

Comment:
Yes, the correction changing $.z to $!z is correct, but you've added the word "a" at the end of the line, and this word is now a duplicate;

Reply:
My bad. I didn't notice it!